hygen/_templates/layer/new/layer.ejs.t

```diff
@@ -7,22 +7,43 @@ import { selectSourceById } from '@carto/react/redux';
 import { useCartoLayerFilterProps } from '@carto/react/api';
 import htmlForFeature from 'utils/htmlForFeature';
 
+export const <%= h.changeCase.constantCase(name) %>_ID = '<%= h.changeCase.camelCase(name) %>';
+
 export default function <%= h.changeCase.pascalCase(name) %>() {
   const { <%= h.changeCase.camelCase(name) %> } = useSelector((state) => state.carto.layers);
   const source = useSelector((state) => selectSourceById(state, <%= h.changeCase.camelCase(name) %>?.source));
   const cartoFilterProps = useCartoLayerFilterProps(source);
 
   if (<%= h.changeCase.camelCase(name) %> && source) {
+    <% if(SQLLayer){ %>
```

Comment:
With latest changes, now code for both layers is the same, so we can remove the IF and the SQLLayer reference